PWGJE/Tasks/jetHadronRecoil.cxx

```diff
@@ -166,8 +161,15 @@ struct hJetAnalysis {
       registry.fill(HIST("hJetPt"), jet.pt());
       registry.fill(HIST("hJetEta"), jet.eta());
       registry.fill(HIST("hJetPhi"), jet.phi());
+      for (auto& jetWTA : jet.template matchedJetGeo_as<std::decay_t<W>>()) {
+        double deltaPhi = RecoDecay::constrainAngle(jetWTA.phi() - jet.phi(), -M_PI);
```

Comment:
```suggestion
        double deltaPhi = RecoDecay::constrainAngle(jetWTA.phi() - jet.phi(), -o2::constants::math::PI);
```
Do not use `M_PI`. It is not a C++ standard and in O2Physics, we have it defined among other constants.